test/utils/create_resources.go

```diff
@@ -27,7 +27,7 @@ import (
 	batch "k8s.io/api/batch/v1"
 	storage "k8s.io/api/storage/v1"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
@alexanderConstantinescu un-necessary change?

Reply:
goimports will complain, updated/old tooling does silly things, saw this on my repo as well.  Just do whatever makes CI happy.

Reply:
OK, let me modify that.